### PR TITLE
airframe-sql: Implement outputAttributes of Unnest

### PR DESCRIPTION
**Pull request by takezoe**

### CODE REVIEW

File: airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala [hunk 1]
```diff
@@ -790,9 +790,14 @@ object LogicalPlan {
 
   case class Unnest(columns: Seq[Expression], withOrdinality: Boolean, nodeLocation: Option[NodeLocation])
       extends Relation {
-    override def children: Seq[LogicalPlan]       = Seq.empty
-    override def inputAttributes: Seq[Attribute]  = Seq.empty // TODO
-    override def outputAttributes: Seq[Attribute] = Seq.empty // TODO
+    override def children: Seq[LogicalPlan]      = Seq.empty
+    override def inputAttributes: Seq[Attribute] = Seq.empty // TODO
```
Comment (xerial) [inline airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala:794]: ok. Filed #2620

File: airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala [hunk 2]
```diff
@@ -790,9 +790,14 @@ object LogicalPlan {
...
+    override def outputAttributes: Seq[Attribute] = {
+      columns.map {
+        case a: ArrayConstructor => SingleColumn(MultiColumn(a.values, None), None, None, a.nodeLocation)
```
Comment (xerial) [inline airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala:797]: Not sure whether we should retain all of the input expressions. I think if we can associate the column name for each ArrayConstructor, we can discard Literal values here except the other Attributes (Id, ResolvedAttributes, etc.).

Comment (takezoe) [reply airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala:797]: That's right. I think the same thing can be applied to SELECT FROM VALUES.

Comment (xerial) [reply airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala:797]: It's ok to merge. We can fix the behavior later.

Comment (takezoe) [reply airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala:797]: Got it. Thank you.

**Pull request merged**